site/app/templates/grading/electronic/Details.twig

```diff
@@ -687,3 +686,16 @@
 {% endmacro %}
 
 {# /Column functions #}
+<script>
+    function changeAnon() {
+        const status = '{{ anon_mode }}';
```

Comment:
Hm, when `anon_mode === true`, does this evaluate in twig to be `const status = '1'` and then for `anon_mode === false`, we get `const status = ''`? If possible, I think I would prefer doing something like  `const status = {{ anon_mode ? 1 : 0 }}` to make what's going on a bit less ambiguous if you don't know how PHP coerces boolean to strings.

Reply:
Ok that makes sense, thanks